api/src/index.ts

```diff
@@ -67,47 +53,10 @@ fastify.get("/", (_, reply) => {
   reply.send(cacheData);
 });
 
-/**
- * Call this route to generate the data again
- * Used by an external cronjob multiple time a day
- */
-fastify.get<{ Querystring: { token: string } }>(
-  "/generate-data",
-  (request, reply) => {
-    const token = request.query && request.query.token;
-
-    if (token === config.TOKEN) {
-      console.log("Request data starting...");
-      generateDataStats()
-        .then(async () => {
-          console.log("Request data generated");
-
-          /**
-           * On production we call a webhook URL that will regenerate the
-           * static client site
-           */
-          if (config.isProduction) {
-            const response = await fetch(config.REBUILD_WEBHOOK_URL, {
-              method: "GET",
-            });
-            const data = await response.json();
-            console.log(`Called webhook`, data);
-          }
-        })
-        .catch((error) => {
-          console.error(error);
-        });
-
-      reply.send({ success: true });
-      return;
-    }
-
-    reply.send({ success: false });
-  }
-);
-
 // Run the server!
 fastify.listen(4000, "0.0.0.0", (err, address) => {
   if (err) throw err;
+  // Every 3rd hour
+  startCron(generateDataStatsQueue, "generate-data-stats", "0 */3 * * *");
```

Comment:
can you move this one after the other cron? that way we are sure the generate data can't be run at the same time